bundles/org.openhab.binding.digitalstrom/README.md

```diff
@@ -394,7 +394,7 @@ _Blinds:_
 
 ## Full Example
 
-### demo.things:
+### demo.things
```

Comment:
From a consistency perspective, is there a reason that you in some cases prefer "### demo.things" and in other cases (such as dmx) prefer "### `demo.things` Example"? In both cases they were previously "### demo things". I might be missing some context, so maybe I just need the explanation. 🙂

Reply:
Yeah, trying to be consistent here, but might have missed some. It should be the latter one, that looks best.